samples/v1/stable.config.testsuite.yaml

```
@@ -12,3 +12,48 @@ tests:
       apiVersion: samples.operator.openshift.io/v1
       kind: Config
       spec: {}
+  - name: Should be able to add a helmchartname to skippedHelmCharts list
+    initial: |
+      apiVersion: samples.operator.openshift.io/v1
+      kind: Config
+      spec: 
+        skippedHelmCharts: 
+        - foo-bar.bar-foo.baz
+    expected: |
+      apiVersion: samples.operator.openshift.io/v1
+      kind: Config
+      spec: 
+        skippedHelmCharts:
+        - foo-bar.bar-foo.baz
+  - name: Should reject invalid string in skippedHelmCharts list
+    initial: |
+      apiVersion: samples.operator.openshift.io/v1
+      kind: Config
+      spec: 
+        skippedHelmCharts:
+        - foo.-bar.baz
+    expectedError: "Invalid value: \"array\": skippedHelmCharts must be a valid Kubernetes resource name. May contain only lowercase alphanumeric characters, hyphens and periods, and each period separated segment must begin and end with an alphanumeric character"
+  - name: Should reject invalid string in skippedHelmCharts list
+    initial: |
+      apiVersion: samples.operator.openshift.io/v1
+      kind: Config
+      spec: 
+        skippedHelmCharts:
+        - foo.bar.baz.
+    expectedError: "Invalid value: \"array\": skippedHelmCharts must be a valid Kubernetes resource name. May contain only lowercase alphanumeric characters, hyphens and periods, and each period separated segment must begin and end with an alphanumeric character"
+  - name: Should reject empty string in skippedHelmCharts list
+    initial: |
+      apiVersion: samples.operator.openshift.io/v1
+      kind: Config
+      spec: 
+        skippedHelmCharts:
+        - 
```

This is actually picking up a different error, can you try this instead please
```suggestion
        - ""
```